### DISCUSSION TITLE
queryIdByfeature

### DISCUSSION THREAD

**Discussion by buptzouy** [milvus-io/milvus#17437] [Q&A and General discussion]
when I execute queryIdbyfeature several times, milvus will always be unloaded. what's wrong?

**Comment by yhmo** [comment]
Which milvus version? Which sdk?
Milvus has no interface named "queryIdbyfeature",  do you mean "get vector by id"?

**Reply by buptzouy** [reply]
> 

Yes, I mean get vector by id. 
Milvus version 2.0.2, 
sdk version 2.0.4

**Reply by yhmo** [reply]
In v2.0, the query() interface can get vector by id. So, your question is: after call query() several times, the target collection was unloaded unexpectedly?

**Reply by xiaofan-luan** [reply]
we do we mean by unloaded?

**Reply by buptzouy** [reply]
> 



> In v2.0, the query() interface can get vector by id. So, your question is: after call query() several times, the target collection was unloaded unexpectedly?

Yes, milvus was unloaded unexpectly

**Reply by yhmo** [reply]
> > 
> 
> > In v2.0, the query() interface can get vector by id. So, your question is: after call query() several times, the target collection was unloaded unexpectedly?
> 
> Yes, milvus was unloaded unexpectly

“milvus was unloaded" ----- What's that mean ？

**Reply by buptzouy** [reply]
> 

![image](https://user-images.githubusercontent.com/3134919/172571020-3d68923a-6063-434a-819e-ef8ccb17db07.png)

**Reply by yhmo** [reply]
Could you send the log of query node to us?
